plugin/fuzz/source/dextool/plugin/fuzz/backend/fuzzvariant.d

```diff
@@ -460,7 +454,7 @@ body {
     import std.string : toLower, indexOf;
     import std.algorithm : endsWith;
 
-    string base_class;
+    string base_class = "";
```

Comment:
As said in previous comment, generateClass() has been rewamped with use of tagging and splitted to three functions instead of one large.